v0/destinations/af/transform.js

```diff
@@ -196,7 +195,7 @@ function processEventTypeTrack(message) {
       isMultiSupport
     );
   }
-  payload.eventName = eventName;
```

Comment:
Just curious, what would happen if `message.event` is not set while sending the payload ?

Reply:
I think it'll fail but it'd have been the scenario previously as well. 
Good point. So, we'll have to think through this while we build a more stable solution as part of this bug fix.
CC @utsabc

Reply:
@yashasvibajpai can we look into this?

Reply:
Sure

Reply:
CC @sanpj2292